qemu/tests/cfg/virtio_driver_sign_check.cfg

```diff
@@ -1,16 +1,26 @@
 - virtio_driver_sign_check:
     virt_test_type = qemu
-    only Win7.x86_64
-    # This script only need run once for every virtio driver version.
-    # So need not add to our testing loop.
+    only Windows
     type = virtio_driver_sign_check
-    cdroms += " sdk virtio"
-    cdrom_virtio = isos/windows/virtio-win.iso
-    cdrom_sdk = isos/windows/GRMSDKX_EN_DVD.iso
+    cdroms += " virtio"
+    cdrom_virtio = isos/windows/virtio-win.latest_prewhql.iso
```

Comment:
Why use virtio-win.latest_prewhql.iso here?
We also need to test this case with whql'ed versions